### PR TITLE
(SIMP-5225) Update for 7.5 and Puppet 5 

### PR DESCRIPTION
**Pull request by jeannegreulich**
- updated package list for 7.5
    - updated packages.yaml
    - edited release mappings
    - temp edits to rpm/dependencies to remove rubygem-puppetserver-toml
      dependancy for repo closure
    - temp update to Puppetfile.tracking to point to updated simp-adapter
      module in  jeannegreulich github.
    
    SIMP-5225 #comment  needs simp-adapter and rubygem-puppetserver* PRs

### CODE REVIEW

File: build/rpm/dependencies.yaml [hunk 1]
```diff
@@ -101,13 +101,13 @@
...
   :obsoletes:
     'pupmod-bfraser-grafana': '2.5.0-2016.1'
-  :external_dependencies:
-    'rubygem-puppetserver-toml':
-      :min: '0.1.2'
   :requires:
     # Exclude the problematic puppet-archive module.
     # ('puppet generate types' fails with puppet-archive).
     - 'pupmod-puppetlabs-stdlib'
+#  :external_dependencies:
```
Comment (jeannegreulich) [inline build/rpm/dependencies.yaml]: Need to uncomment these out when SIMP-5325 is fixed.

File: spec/acceptance/suites/rpm_docker/nodesets/el7.yml [hunk 2]
```diff
@@ -74,7 +74,8 @@ HOSTS:
       - 'yum --enablerepo=base -y install python27'
 
       # RVM
-      - 'runuser build_user -l -c "gpg2 --keyserver hkp://pgp.mit.edu --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB || gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB"'
+      #      - 'runuser build_user -l -c "gpg2 --keyserver hkp://pgp.mit.edu --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB || gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB"'
+      - 'runuser build_user -l -c "gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB || gpg2 --keyserver hkp://pgp.mit.edu --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB "'
```
Comment (jeannegreulich) [inline spec/acceptance/suites/rpm_docker/nodesets/el7.yml]: Keep getting RPG download failure on the first one and it does not go to the second.

File: Puppetfile.tracking [hunk 3]
```diff
@@ -17,8 +17,10 @@ mod 'simp-rsync_data',
   :ref => '6.2.1'
 
 mod 'simp-adapter',
-  :git => 'https://github.com/simp/simp-adapter',
-  :ref => '0.0.6'
+#  :git => 'https://github.com/simp/simp-adapter',
+#  :ref => '0.1.0'
+  :git => 'https://github.com/jeannegreulich/simp-adapter',
```
Comment (jeannegreulich) [inline Puppetfile.tracking]: Need to set this back to master branch on simp github once simp-adapter changes have been  been pushed.

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 4]
```diff
@@ -148,45 +127,21 @@ caribou-gtk2-module
 caribou-gtk3-module
 ca-certificates
 ccid
-ccs
-cdi-api
 cdparanoia-libs
 centos-bookmarks
 centos-indexhtml
 centos-logos
-centos-lsb
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt]: Don't we have some facts that use lsb?

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 5]
```diff
@@ -219,8 +169,6 @@ cronie-anacron
 crontabs
 cryptsetup
 cryptsetup-libs
-cryptsetup-luks
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt]: Do we need this for encrypting disk?

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 6]
```diff
@@ -299,11 +237,8 @@ ebtables
 ecj
 ed
 efibootmgr
-efivar-devel
 efivar-libs
-eggdbus
 elasticsearch
-elasticsearch-curator
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt]: We use elasticsearch-curator.  Why was this removed?

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 7]
```diff
@@ -403,60 +331,35 @@ glib-networking
...
 gnutls-dane
 gnutls-utils
 gobject-introspection
-google-guice
 gperftools-libs
 gpgme
-gpg-pubkey
 gpm
 gpm-libs
-grafana
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt]: Why remove grafana if you have kept elasticsearch?

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 8]
```diff
@@ -1850,17 +1493,121 @@ yum
...
+python2-oauthlib
+python2-pyasn1
+python2-pyasn1-modules
+lz4
+qemu-guest-agent
+mod_nss
+hesiod
+libbluray
+
+# Packages not on the ISO but downloaded using packages.yaml
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt]: Are these needed?

File: build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt [hunk 9]
```diff
@@ -1850,17 +1493,121 @@ yum
...
 yum-plugin-versionlock
-yum-presto
-yum-rhn-plugin
 yum-utils
 zd1211-firmware
 zenity
 zip
 zlib
 zlib-devel
+authconfig
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt]: why was this moved down?  It is much easier to understand this package list if it is alphabetical

File: build/distributions/CentOS/7/x86_64/release_mappings.yaml [hunk 10]
```diff
@@ -6,19 +6,36 @@ simp_releases:
...
+        build_command: 'bundle exec rake build:auto[CentOS-7-x86_64-DVD-1804.iso,6.X]'
+        os_version:   '7.0'
+
+      RedHat:
+        isos:
+          - name:     'rhel-server-7.5-x86_64-dvd.iso'
+            size:     4059037696
+            checksum: 'd0dd6ae5e001fb050dafefdfd871e7e648b147fb2d35f0e106e0b34a0163e8f5'
+        build_command: 'bundle exec rake build:auto[rhel-server-7.5-x86_64-dvd.iso,6.X]'
+        os_version:   '7.5'
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/release_mappings.yaml]: Should we have entries for 6.1.0 and 6.2.0?

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml.orig [hunk 11]
```diff
@@ -0,0 +1,274 @@
+---
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml.orig]: Why was this file added?
